CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## Developer Changelog
+
 ### May 2, 2022 DESCW-280
 * frontend
   * added new header, footer and sidebar
+* Add CODEOWNERS file.
```

Comment:
I would just be careful about tacking on unrelated work when you've got the ticket number as the heading. Not a huge deal but I think it hurts our ability to track what work was done when.